lib/modules/dosomething/dosomething_helpers/dosomething_helpers.module

```diff
@@ -189,6 +189,9 @@ function dosomething_helpers_extract_field_data($field, $language = LANGUAGE_NON
         // Date values, so need to extract using custom method.
         $values[] = dosomething_helpers_extract_field_dates($content);
       }
+      else if (in_array('country', $keys)) {
+        $values[] = $content['country'];
```

Comment:
I think this should be good for now, but add a `@TODO` or some comment to help explain what's going on and maybe that this needs a dedicated function later on (like `dosomething_helpers_extract_field_location_data()` or something) for extracting all the info at a later date.